Release RC2 (#271)
JoshOrndorff/recipes: text/1-prepare-kitchen/1-build-node.md

cargo build --release -p kitchen-node
```

As you work through the recipes, refer back to these instructions each time you wish to re-compile
the node. Over time the commands will become familiar, and you will even modify them to compile
other nodes.

## Checking Your Work

> Once the compilation is completed, you can ensure that the node has built properly by displaying its

Reviewer: ```suggestion
Once the compilation is completed, you can ensure that the node has been built properly by displaying its
```

> help page. Notice that the node has built to the `target/release` directory. This is the default

Reviewer: ```suggestion
help page. Notice that the node executable is found in the `target/release` directory. This is the default
```
